docs/api.rst

```diff
@@ -264,9 +281,7 @@ throwError
 
   throwError :: Error -> Stream void
 
-Create a :ref:`Stream` that fails at time 0 with the provided ``Error``.
-
-This can be useful for functions that need to return a :ref:`Stream` and also need to propagate an error. ::
+Create a :ref:`Stream` that fails at time 0 with the provided ``Error``.  This can be useful for functions that need to return a :ref:`Stream` and also need to propagate an error. ::
```

Comment:
I would switch the order, because it's the `Stream` that fails with the `Error`:

> Create a :ref:`Stream` that fails with the provided \`\`Error\`\` at time 0.